GUI/src/main/kotlin/interface/FrameNavigationInterface.kt

```diff
@@ -0,0 +1,30 @@
+
+/**
+ * Provides navigation methods for frames in a video.
+ *
+ * This interface defines methods to navigate frames in various ways, including jumping a specified number
+ * of frames in a given direction, jumping to a specific percentage of the video, jumping to a specific frame,
+ * and jumping to the next difference frame. Implementing classes should provide implementation for these methods.
+ */
+interface FrameNavigationInterface {
+
+    /**
+     * Jump n frames in a specified direction
+     */
+    fun jumpFrames(frames: Int, forward: Boolean)
```

Comment:
boolean is unnecessary, since frames can be negative

Reply:
I thought that an additional boolean would be more clear, but sure I can remove it.